pgcli/main.py

```diff
@@ -75,9 +75,24 @@
     ])
 MetaQuery.__new__.__defaults__ = ('', False, 0, False, False, False, False)
 
-
 class PGCli(object):
 
+    def set_default_pager(self, config):
+        configured_pager = config['main'].get('pager')
+        os_environ_pager = os.environ.get('PAGER')
+
+        if configured_pager:
+            self.logger.info('Default pager found in config file: ' + '\'' + configured_pager + '\'')
+            os.environ['PAGER'] = configured_pager
+        elif (os_environ_pager):
+            self.logger.info('Default pager found in PAGER environment variable: ' + '\'' + os_environ_pager + '\'')
+            os.environ['PAGER'] = os_environ_pager
+        else:
+            self.logger.info('No default pager found in environment. Using os default pager')
+        # Always set default set of less recommended options, they are ignored if pager is 
+        # different than less or is already parameterized with their own arguments 
+        os.environ['LESS'] = '-SRXF'
+
```

Comment:
An alternate succinct implementation: 

```
def set_default_pager(self, config):
    os.environ['PAGER'] = os.environ.get('PAGER') or config['main'].get('pager')
```

I want to keep the default pager uncommented in the config file. That way if the user has setup a PAGER then we don't mess with it, but if they haven't then we give them a sane value for that option. This gives a more pleasant experience out of the box.

Also this new change might break existing pgcli behavior for people who already have pgcli installed.